neuralmonkey/learning_utils.py

```diff
@@ -300,6 +301,9 @@ def training_loop(tf_manager: TensorFlowManager,
 
     log("Finished.")
 
+    if interrupt is not None:
+        raise interrupt  # pylint: disable=raising-bad-type
```

Comment:
když jsem googlil, tak `raise interrupt()` prej funguje a nevyhazuje pylint errory. Stejně tak bych možná zkusil hodit `KeyboardInterrupt(interrupt)`

Reply:
`raise interrupt()` mi nefunguje a podle mě by si to stěžovalo, že `NoneType` (ani `KeyboardInterrupt`) není callable. To druhé zase vyrobí dva tracebacky.

Zřejmě je to [vada pylintu](https://stackoverflow.com/questions/2228790/why-does-pylint-give-error-e0702-raising-nonetype-on-this-raise-statement), tak bych to řešil vypnutím pylintu.

Reply:
hmm.. a co zavolat `typing.cast`? (https://docs.python.org/3/library/typing.html#typing.cast)

Reply:
Ale je fakt, že to nemusí bejt...